ethereum/contracts/TablelandRigs.sol

```diff
@@ -507,6 +545,25 @@ contract TablelandRigs is
         }
     }
 
+    /**
+     * @dev See {ITablelandRigs-parkRigAsParkingAdmin}.
+     */
+    function parkRigAsParkingAdmin(uint256[] calldata tokenIds) external onlyParkingAdmin {
```

Comment:
To save contract size, looks like you could make a private `_parkAsOwnerOrAdmin` method and call it from `parkRigAsParkingAdmin` and `parkRigAsOwner`, which are gated by respective modifiers.

Reply:
Alternatively, you could have a modifier that is `onlyOwnerOrAdmin`, and a single external method `parkRigAsOwnerOrAdmin` (I don't think we need to worry about breaking changes for owner only methods).

Reply:
I added a private function that is called from both the Owner and Admin park functions